braze/build.gradle

```
@@ -27,9 +27,9 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar', '*.aar'])
 
     // RudderStack SDK
-    implementation 'com.rudderstack.android.sdk:core:[1.0,2.0)'
+    implementation 'com.rudderstack.android.sdk:core:[1.12,2.0)'
```

```suggestion
    compileOnly 'com.rudderstack.android.sdk:core:[1.12,2.0)'
```

Reason: `compileOnly` ensures the library is present during compile time but it’s unavailable during runtime. Since it’s mandatory for the library users to have RudderStack core SDK in their apps as a direct dependency, hence we can safely assume that the library will be present during runtime.